baseapp-chats/baseapp_chats/permissions.py

```
@@ -3,45 +3,82 @@
+                models.Q(actor_id=current_profile.id, target_id__in=participant_profile_ids)
+                | models.Q(actor_id__in=participant_profile_ids, target_id=current_profile.id)
+            )
+
+            if blocks_qs.exists():
+                return False
+
+            return True
+
+    def has_perm(self, user_obj, perm, obj=None):
+        if perm == "baseapp_chats.add_chatroom" and user_obj.is_authenticated:
+            return can_add_chatroom(user_obj, obj)
```

Ruff (0.8.2)

Undefined name `can_add_chatroom`

(F821)